server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -8,17 +28,99 @@ export PB_ELASTICSEARCH_IMAGE_TAG=${PB_ELASTICSEARCH_IMAGE_TAG:-"7.13.3"}
 export PB_POSTGRESQL_IMAGE_PULL_POLICY="${PB_POSTGRESQL_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 export PB_KEYCLOAK_IMAGE_PULL_POLICY="${PB_KEYCLOAK_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 export PB_SERVER_IMAGE_PULL_POLICY="${PB_SERVER_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
 
 export PB_DASHBOARD_DIR="${PB_DASHBOARD_DIR:-${PWD}/dashboard/build/}"
 
-envsubst < $(dirname ${0})/pod.yml | podman play kube -
+podman pod create --name ${PB_POD_NAME} --hostname pbenchinacan -p 9200:9200 -p 5432:5432 -p 8090:8090 -p 8080:8080
+
+podman run \
+  --rm \
+  --name "${PB_POD_NAME}-elasticsearch" \
+  --pod ${PB_POD_NAME} \
+  --workdir /usr/share/elasticsearch \
+  --pull ${PB_ELASTICSEARCH_IMAGE_PULL_POLICY} \
+  --umask 0000 \
+  --restart no \
+  --pids-limit 0 \
+  -e PATH=/usr/share/elasticsearch/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
+  -e HOME=/root \
+  -e TERM=xterm \
+  -e container=oci \
+  -e discovery.type=single-node \
+  -e ES_JAVA_OPTS="-Xms1g -Xmx1g"\
+  -e xpack.security.enabled=false \
+  -e ES_PORT=9200 \
+  -d \
+  ${PB_ELASTICSEARCH_IMAGE_NAME}:${PB_ELASTICSEARCH_IMAGE_TAG} \
+  /bin/tini -- /usr/local/bin/docker-entrypoint.sh eswrapper
+
+
+podman run \
+  --rm \
+  --pod ${PB_POD_NAME} \
+	--name "${PB_POD_NAME}-postgres" \
+	--workdir /opt/app-root/src \
+	--pull ${PB_POSTGRESQL_IMAGE_PULL_POLICY} \
+	--umask 0000 \
+	--user 26:26 \
+	--restart no \
+  --pids-limit 0 \
+  --entrypoint container-entrypoint \
+	-e POSTGRESQL_USER=pbenchcontainer \
+	-e POSTGRESQL_PASSWORD=pbench \
+	-e POSTGRESQL_DATABASE=pbenchcontainer \
+	-e HOME=/var/lib/pgsql \
+	-e CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/postgresql \
+	-e APP_ROOT=/opt/app-root \
+	-e POSTGRESQL_VERSION="13" \
+	-e PGUSER=postgres \
+	-e STI_SCRIPTS_URL=image:///usr/libexec/s2i \
+	-d \
+	${PB_CONTAINER_REG}/${PB_ORG_NAME}/${PB_POSTGRESQL_IMAGE_NAME}:${PB_POSTGRESQL_IMAGE_TAG} \
+	run-postgresql -p 5432
+
+podman run \
+  --rm \
+  --pod ${PB_POD_NAME} \
+  --name "${PB_POD_NAME}-keycloak" \
+  --pull ${PB_KEYCLOAK_IMAGE_PULL_POLICY} \
+  --umask 0000 \
+  --restart no \
+  --pids-limit 0 \
+  -e KEYCLOAK_ADMIN=admin \
+  -e KEYCLOAK_ADMIN_PASSWORD=admin \
+  -d \
+  quay.io/keycloak/keycloak:latest \
+  start-dev --health-enabled=true --http-port=8090
+
+sh server/pbenchinacan/load_keycloak.sh
+
+# Create a dashboard src directory to mount on pbench-server container's
+# dashboard directory
+mkdir -m 0755 -p ${PB_DASHBOARD_DIR}
+
+podman run \
+  --rm \
+  --pod ${PB_POD_NAME} \
+  --name "${PB_POD_NAME}-pbenchserver" \
+  --pull ${PB_SERVER_IMAGE_PULL_POLICY} \
+  --umask 0000 \
+  --restart no \
+  --pids-limit 0 \
+  --entrypoint /sbin/init \
+  --workdir / \
+  -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
+  -v ${PB_DASHBOARD_DIR}:/srv/pbench/public_html/dashboard:z \
+  -d \
+  ${PB_CONTAINER_REG}/${PB_ORG_NAME}/${PB_SERVER_IMAGE_NAME}:${PB_SERVER_IMAGE_TAG} \
+
```

Comment:
Since nothing follows line 125, you should omit its trailing slash.

And, with that gone, you can probably remove line 126, as well.